docs/articles/documentation/extending-testcafe/reporter-plugin/helpers.md

```diff
@@ -95,7 +95,7 @@ Parameter | Type    | Description
 The following example demonstrates how to enable word wrapping in case the `title` string is long.
```

Comment:
The following example demonstrates how to enable word wrapping when the `title` string is long: